rero_ils/modules/files/cli.py

```diff
@@ -58,7 +58,7 @@ def create_pdf_file(document):
     return generator.output()
 
 
-def create_pdf_record_files(document, metadata, flush=False):
+def create_pdf_record_files(document, metadata, flush=False, n=None):
```

Comment:
Could you give more information for `n`. Minimum recommendation for variable/parameter length is 3! Missing in doc string `flush`, `n`.